lib/ProductOpener/KnowledgePanels.pm

```diff
@@ -822,8 +822,11 @@ sub create_health_card_panel ($product_ref, $target_lc, $target_cc, $options_ref
 
 	$log->debug("create health card panel", {code => $product_ref->{code}}) if $log->is_debug();
 
-	create_nutriscore_panel($product_ref, $target_lc, $target_cc, $options_ref);
-	if ($options_ref->{admin} || $options_ref->{moderator} || $options_ref->{producers_platform}) {
+	if ($target_cc eq "fr") {
```

Comment:
We do , especially since it is now a canary for the mobile app

Reply:
We can leave it for now, but it's going to create confusion going forward. We probably should update the old score logo with an "old computation" or something so that it's clear to producers.